pkg/controller/flow/reconcile.go

```diff
@@ -266,26 +261,22 @@ func (r *reconciler) createSubscription(channelName string, target string, flow
 }
 
 func (r *reconciler) reconcileFeed(channelDNS string, flow *v1alpha1.Flow) (*feedsv1alpha1.Feed, error) {
-	feedName := flow.Name
-
-	feed := &feedsv1alpha1.Feed{}
-	err := r.client.Get(context.TODO(), client.ObjectKey{Namespace: flow.Namespace, Name: feedName}, feed)
+	feed, err := r.getFeedForFlow(flow)
 	if errors.IsNotFound(err) {
 		feed, err = r.createFeed(channelDNS, flow)
 		if err != nil {
-			glog.Errorf("Failed to create feed %q : %v", feedName, err)
+			glog.Errorf("Failed to create feed: %v", err)
```

Comment:
Can we keep the flow name in this log message but properly label that information? ("Failed to create feed for flow %v: %v", flow.Name, err)

Reply:
Done.